src/FilterList.js

```diff
@@ -23,6 +26,23 @@ export let Label = inject(_.pick('tree'))(
       {tree &&
         node && (
           <span className="filter-field-label-icon">
+            {!node.updating &&
+              !!node.hasValue &&
```

Comment:
It shouldn’t matter if the current node has a value

Reply:
removed the hasValue check